sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -287,6 +289,42 @@ abstract class JdbcDialect extends Serializable with Logging{
     s"COMMENT ON SCHEMA ${quoteIdentifier(schema)} IS NULL"
   }
 
+  /**
+   * Creates an index.
+   *
+   * @param indexName         the name of the index to be created
+   * @param indexType         the IndexType of the index to be created
```

Comment:
Why do we capitalize like `the IndexType`? I expected something like the following.
```
- @param indexType         the IndexType of the index to be created
+ @param indexType         the type of the index to be created
```

Reply:
Done